### ISSUE TITLE
Add nickname to the project card

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#4564]
As a tC user I would like to be able to see the nickname I assigned to the project without having to open the project details screen or selecting it, so that it's easier to find the project I want to work on.

## Specs
- [ ] On the project card
  - [ ] Add the Nickname as subtitle
- [ ] Show the Nickname as a hover-over in the Setup Stepper 
- [ ] Show the Nickname as a hover-over in the top menu bar

## Mockups
![image](https://user-images.githubusercontent.com/13352518/46228622-3fb90b80-c328-11e8-920d-85106a296f4f.png)

![image](https://user-images.githubusercontent.com/13352518/46228607-2f089580-c328-11e8-9771-153c0523bece.png)

**Comment by lrsallee** [comment]
I see where this is going, but an underwhelmed. I would indeed like to see the project nickname. But I can see that when I hover over the project name in tC 1.0.0. I have been providing enough info in the nicknames to **supplement** the booknames, but not to **replace** them. To me, adding the hover name is a sufficient solution. If we swap the positions like this, I will need to go back and enter all new nicknames. I can do that, but it is a pain. Also, I suspect that a lot of users will not be very good at creating descriptive nicknames. In fact, I suspect that their choices will often be less helpful than the generated book names.

**Comment by pjoakes** [comment]
Yeah, I think the official name should be primary, with the nickname appearing as a subtitle or hover-over.

**Comment by russp41** [comment]
I have a different use case. I would like the option to sort by the nickname on the project card instead of the generated name. Names that begin with "kn-x-handijog..." and "lmn..." and "kpb..." (all examples in the PMN network) are not very helpful if you have multiple projects, e.g., if you are a project manager. I gave nicknames beginning with "PMN" and it would be helpful to be able to sort the list by those. A project manager in the PMN network where there are dozens of language projects might like to standardize nicknames across projects, maybe using state names or something else. One suggestion would be to show the generate name, nickname, language, book in columns on the project card, then allowing the user to click the column header to sort by column.  However, maybe stepping back further and thinking how a project manager or quality checker might use tC when they are involved in multiple projects (there are 70+ language projects in the PMN network) needs to be considered. Having all NT books loaded for a few dozen projects would get unwieldy in a hurry. Maybe filtering of the project card list needs to be considered, but again, a look at how people in these roles will use tC is needed.
